app/src/views/public/public-view.vue

```diff
@@ -10,7 +10,7 @@
 				</div>
 				<div class="title">
 					<h1 class="type-title">{{ branding && branding.project_name }}</h1>
-					<p class="subtitle">Admin App</p>
+					<p class="subtitle">Application</p>
```

Comment:
We should use translations for this

Reply:
I actually would like to make this configurable in Settings, and add it to the private view too (below the Project Name in the nav). That way you can have:

```
Monospace Inc
Development
```

Or...

```
Monospace Inc
Data Portal
```